src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Mappers/DraftApprenticeshipTests/WhenIMapViewDraftApprenticeshipDetailsToViewModel.cs

```diff
@@ -94,9 +100,9 @@ public async Task ThenDateOfBirthIsMappedCorrectly()
             Assert.AreEqual(_apiResponse.DateOfBirth, result.DateOfBirth);
         }
 
-        [TestCase(DeliveryModel.Regular)]
-        [TestCase(DeliveryModel.PortableFlexiJob)]
-        public async Task ThenDeliveryModelIsMappedCorrectly(DeliveryModel dm)
+        [TestCase(Infrastructure.OuterApi.Types.DeliveryModel.Regular)]
+        [TestCase(Infrastructure.OuterApi.Types.DeliveryModel.PortableFlexiJob)]
+        public async Task ThenDeliveryModelIsMappedCorrectly(Infrastructure.OuterApi.Types.DeliveryModel dm)
```

Comment:
a possibility is to use this in the using statement area

using DeliveryModel = SFA.DAS.ProviderCommitments.Infrastructure.OuterApis.Types.DeliveryModel;

to reduce the noise, but it's up to you

Reply:
Initially i did something similar, but as we are also including the commitments nuget types VS has a hard time working out which ones to use. Decided to be safe here with full namespace